pkg/ingester/client/mimir_util_test.go

```diff
@@ -79,7 +78,7 @@ func TestSendQueryStream(t *testing.T) {
 	// Try to receive the response and assert the error we get is the context.Canceled
 	// wrapped within a gRPC error.
 	_, err = stream.Recv()
-	s, ok := status.FromError(err)
+	s, ok := grpcutil.ErrorToStatus(err)
```

Comment:
[nit]
Another possibility would be:

```suggestion
	code := grpcutil.ErrorToStatusCode(err)
```

and then to check 
```
require.Equal(t, codes.Canceled, code)
```